src/storage/btree.c

```diff
@@ -22459,6 +22459,38 @@ btree_rv_data_pack_btid_and_stats (const BTID * btid, long long nulls, long long
   written_size = CAST_BUFLEN (datap - data);
 }
 
+/*
+ * btree_rv_global_unique_stats_commit_dump - dump stored info about global unique stats
+ *
+ */
+void
+btree_rv_global_unique_stats_commit_dump (FILE * fp, int length, void *data)
+{
+  BTID btid;
+  log_unique_stats stats;
+
+  assert (length >= (3 * OR_BIGINT_SIZE) + OR_BTID_ALIGNED_SIZE);
+
+  const char *data_ptr = (const char *) data;
+
+  OR_GET_BTID (data_ptr, &btid);
+  data_ptr += OR_BTID_ALIGNED_SIZE;
+
+  OR_GET_BIGINT (data_ptr, &stats.num_nulls);
+  data_ptr += OR_BIGINT_SIZE;
+
+  OR_GET_BIGINT (data_ptr, &stats.num_oids);
+  data_ptr += OR_BIGINT_SIZE;
+
+  OR_GET_BIGINT (data_ptr, &stats.num_keys);
+  data_ptr += OR_BIGINT_SIZE;
+
+  assert (CAST_BUFLEN (data_ptr - (const char *)data) == length);
```

Comment:
Why don't you use `btree_rv_data_unpack_btid_and_stats`?

Reply:
oh.. we don't have LOG_RCV in dummy functions... OK.  But isn't it better to create a LOG_RCV arbitrarily to make use of the existing function?

Reply:
Thanks, I will do this.
When I get back to this issues. :)